cloudconfig/podcfg/podcfg.go

```diff
@@ -226,13 +227,27 @@ func (cfg *ControllerPodConfig) GetControllerImagePath() string {
 	return GetJujuOCIImagePath(cfg.Controller.Config, cfg.JujuVersion)
 }
 
-// GetJujuOCIImagePath returns jujud oci image path.
+// GetJUjuDbOCIImagePath returns the juju-db oci image path.
+func (cfg *ControllerPodConfig) GetJujuDbOCIImagePath() string {
+	imageRepo := cfg.Controller.Config.CAASImageRepo()
+	if imageRepo == "" {
+		imageRepo = jujudOCINamespace
+	}
+	return fmt.Sprintf("%s/%s", imageRepo, jujudbOCIName)
```

Comment:
here as well, so should the user be able to specify the mongo version to use in controller config?
`jujudbOCIName` should be just the fallback default.